### ISSUE TITLE
Feature request: Algorithm/Coin/Miner weighting % or preference

### ISSUE THREAD

**Issue by bbgobie** [MultiPoolMiner/MultiPoolMiner#861]
I end up with a bunch of similar estimated profits for different coins.
It would be awesome if say you could give each algo or coin some kind of weighting.
Assuming default is 1, but you are not happy with LyraZ2 seeming to not be as profitable as suggested you could always give it a .8 weighting, or essentially a manual user intervention handicap to the profitability calculation.

Similarly if you don't believe in a coin too much but one you like is usually 2nd or 3rd in your profitability, instead of moving the first two completely, I'd much rather give my coin of choice a little more weight. Eg 1.1 * profitability.  So I might still mine the other 2 if they were much more profitable, but when they were withing 10%, mine my HODL coin.

**Comment by UselessGuru** [comment]
Good idea, but a very advanced feature. This weighting would have to be configured in each miner file. So you are not very far away to do this yourself:

Just add another code block that reduces the hashrate by algo factor X.
